utils/scripts/ocm/ocm.py

```diff
@@ -777,6 +804,52 @@ def wait_for_osd_cluster_to_get_deleted(self, timeout=3600):
             required=True)
         install_rhods_parser.set_defaults(func=ocm_obj.install_rhods_addon)
 
+        #Argument parsers for install_rhods_addon
+        install_gpu_parser = subparsers.add_parser(
+            'install_gpu_addon',
+            help=("Install gpu addon cluster."),
+            formatter_class=argparse.ArgumentDefaultsHelpFormatter)
+        required_install_gpu_parser = install_gpu_parser.add_argument_group('required arguments')
+
+        required_install_gpu_parser.add_argument("--cluster-name",
+            help="osd cluster name",
+            action="store", dest="cluster_name",
+            required=True)
+        install_gpu_parser.set_defaults(func=ocm_obj.install_gpu_addon)
+
+        #Argument parsers for create_cluster
+        add_machinepool_parser = subparsers.add_parser(
+            'add_machine_pool',
+            help=("Adds machine pool to given cluster via OCM."),
+            formatter_class=argparse.ArgumentDefaultsHelpFormatter)
+
+        optional_machinepool_cluster_parser = add_machinepool_parser._action_groups.pop()
+        required_machinepool_cluster_parser = add_machinepool_parser.add_argument_group('required arguments')
+        add_machinepool_parser._action_groups.append(optional_machinepool_cluster_parser)
+
+        required_machinepool_cluster_parser.add_argument("--cluster-name",
+            help="osd cluster name",
+            action="store", dest="cluster_name",
+            required=True)
+
+        optional_machinepool_cluster_parser.add_argument("--instance-type",
+            help="Machine pool instance type",
+            action="store", dest="pool_instance_type", metavar="",
+            default="g4dn.xlarge")
+        optional_machinepool_cluster_parser.add_argument("--replicas",
+            help="Machine pool replicas",
+            action="store", dest="pool_replicas", metavar="",
+            default="1")
+        optional_machinepool_cluster_parser.add_argument("--taints",
+            help="Machine pool taints information",
+            action="store", dest="taints", metavar="",
+            default="nvidia.com/gpu=NONE:NoSchedule")
```

Comment:
Are you sure the `NONE` is going to be converted to an empty string for the value of the taint?

Reply:
I've tested this manually. The `NONE` is not converted but the value of the taint doesn't matter, it could be empty or not and the taint would still work. I think we can keep it like this.